bibliothecary.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "commander"
   spec.add_dependency "deb_control"
+  spec.add_dependency "json", "~> 2.8"
```

Comment:
The ruby json implementation added the ability to accept json with trailing commas in [version 2.8.0](https://github.com/ruby/json/releases/tag/v2.8.0). The `bun.lock` file uses trailing commas in their output and in order to read their file correctly we need this updated version of the json gem that is usually bundled with ruby.